tools/msvs/msi/nodemsi.wixproj

```diff
@@ -72,7 +72,10 @@
     </HeatDirectory>
   </Target>
   <PropertyGroup>
-    <PostBuildEvent>move "!(TargetPath)" "$(TargetDir)\$(TargetFileName)"
-move "!(TargetPdbPath)" "$(TargetDir)\$(TargetPdbName)"</PostBuildEvent>
+    <PostBuildEvent>
+      "$(WixToolPath)\torch.exe" -t language "$(TargetDir)en-US\$(TargetFileName)" "$(TargetDir)de-de\$(TargetFileName)" -out "$(TargetDir)transforms\de-de.mst"
```

Comment:
It compares the en-us installer with the de-de version and extract the language differences into a transformation file.